spec/factories/note.rb

```diff
@@ -2,6 +2,7 @@
   factory :note do
     application_choice
     provider_user
+    user { create(:provider_user) }
```

Comment:
can we add some randomness to this and generate notes with both provider_users and vendor_api users.